logging/loggeroption.go

```diff
@@ -187,3 +187,11 @@ type redirectOutputOption struct {
 func (o *redirectOutputOption) set(l *Logger) {
 	l.redirectOutputWriter = o.writer
 }
+
+// MinimalSeverity is the allowed severity of log entries.
+// If the log entry has the severity lesser than the minimal severity, then the log entry will be ignored.
+func MinimalSeverity(s Severity) LoggerOption { return minimalSeverity(s) }
```

Comment:
nit: I'd suggest to make the name more verbose. "MinimalSeverity" requires additional clarification. Consider "IngestedSeverityThreshold" or "MinimalLoggedSeverity" or something similar.

Reply:
Ok. I will rename `MinimalSeverity` to `MinimalLoggedSeverity`.

Reply:
Done